components/TPremium.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div
+    class="relative top-10 max-w-sm pt-3 pb-4 font-sans tracking-tight mx-auto"
+  >
+    <section class="bg-gray-200 p-4 rounded-lg mb-3">
+      <div class="m-auto w-fit"><img src="img/gift.png" /></div>
+      <h1 class="text-3xl text-center font-extrabold">Try Premium</h1>
+      <p class="text-lg text-center py-3">
+        Invite 5 friends to get a card from your profile and get premium for
+        free for 1 month. Limited offer, 200 left!
+      </p>
+    </section>
+
+    <TBenefits
+      title="Why get Premium?"
+      icon="check-green"
```

Comment:
if it is already set by default, no need to overwrite it

Reply:
You are right, I had changed code